MLS.Agent.Tools/RelativeDirectoryPath.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace MLS.Agent.Tools
 {
-    public class RelativeDirectoryPath :
+    public sealed class RelativeDirectoryPath :
```

Comment:
Your're right, no need to enforce seal here. Just wanted to assert that no unknown direct subclass of RelativePath is created (because we switch on RelativeFilePath and RelativeDirectoryPath), but this is already asserted by the internal constructor of RelativePath.